pipelines/pulpcore/02-install.yaml

```diff
@@ -3,6 +3,9 @@
   hosts:
    - "{{ forklift_server_name }}"
   become: True
+  vars:
+    pulp_pkg_undeclared_deps: '{}'
```

Comment:
a string with `{}` in it? why

Reply:
I thought that I could just empty the list `{}`

Reply:
You can, but then it needs to be an empty list (`[]`) or an empty dictionary (`{}`), not a string `'{}'`